Solution/iOS/RocketAppUITests/Screens/RocketListScreen.swift

```diff
@@ -0,0 +1,48 @@
+import Login
+import RocketList
+import UIToolkit
+import XCTest
+
+struct RocketListScreen: Screen {
+  let app: XCUIApplication
+
+  private let rocketCells: XCUIElementQuery
+  private let rocketNames: XCUIElementQuery
+  private let rocketCellArrows: XCUIElementQuery
+  
+  init(app: XCUIApplication) {
+    self.app = app
+    
+    rocketCells = app.buttons.matching(identifier: AccessibilityKeys.RocketList.rocketCell)
+    rocketNames = app.staticTexts.matching(identifier: AccessibilityKeys.RocketListCell.rocketNameLabel)
+    rocketCellArrows = app.images.matching(identifier: AccessibilityKeys.RocketListCell.rocketCellArrow)
+  }
+
+  @discardableResult
+  func countRocketCells() -> Self {
+    XCTAssert(rocketCells.firstMatch.waitForExistence(timeout: 5), "Rocket cells did not appear within the timeout.")
+    print("Number of rocket cells displayed: \(rocketCells.count)")
+    XCTAssert(rocketCells.count == 4, "Expected exactly 4 rocket cells to be displayed.")
+    return self
+  }
+  
+  @discardableResult
+  func checkRocketNames(_ expectedRocketNames: [String]) -> Self {
```

Comment:
Zacala by som test tym ze rocketNames ma stejny pocet polozek jako expectedRocketNames.
Zosortit obe tie polia podla abecedy.

a pak nieco na styl

```
zip(rocketNames, expectedRocketNames).forEach { name, expectedName
    XCTAssert ze se to rovna
}
```

neskusala som to zbuildit, mozno sa s tym budes musiet trochu pohrat, tady je dokumentace na zip https://developer.apple.com/documentation/swift/zip(_:_:)

Reply:
otazka jestli to vubec testovat, protoze je to public API, ktore sa moze kedykolvek zmenit a pak ti to bude zbytocne failovat